plugins/inputs/bigbluebutton/README.md

```diff
@@ -0,0 +1,44 @@
+# BigBlueButton Input Plugin
+
+The BigBlueButton Input Plugin gathers metrics from [BigBlueButton](https://bigbluebutton.org/) server. It uses [BigBlueButton API](https://docs.bigbluebutton.org/dev/api.html) getMeetings and getRecordings endpoints to query the data.
+
+## Configuration
+
+```toml
+[[inputs.bigbluebutton]]
+	## Required BigBlueButton server url
+	url = "http://localhost:8090"
+
+	## Required BigBlueButton api endpoint
+	api_endpoint = "/bigbluebutton/api/"
+
+	## Required BigBlueButton secret key
+	# secret_key =
+
+	## Server name. Used as "server" tag if not empty. Otherwise use url
+	# server_name = 
```

Comment:
I think you should leave this out and use an existing `tags` section ([see docu](https://github.com/influxdata/telegraf/blob/master/docs/CONFIGURATION.md#input-plugins)) instead.